seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/java/org/apache/seatunnel/e2e/connector/file/local/LocalFileIT.java

```diff
@@ -38,9 +38,9 @@
 import java.nio.file.Paths;
 
 @DisabledOnContainer(
-        value = {TestContainerId.SPARK_2_4},
+        value = {TestContainerId.SPARK_2_4, TestContainerId.FLINK_1_15},
         type = {},
-        disabledReason = "The apache-compress version is not compatible with apache-poi")
+        disabledReason = "The apache-compress version is not compatible with apache-poi, Flink 1.15.3's SideOutputDataStream.class conflict with Flink 1.16.2's")
```

Comment:
Please remove DisabledOnContainer on `TestContainerId.FLINK_1_15`. I believe the changes won't only take effect on certain versions.

Reply:
Roger